xbmc/music/Song.h

```diff
@@ -195,6 +206,7 @@ class CSong final : public ISerializable
   std::string strRecordLabel; // Record label from tag for album processing by CMusicInfoScanner::FileItemsToAlbums
   std::string strAlbumType; // (Musicbrainz release type) album type from tag for album processing by CMusicInfoScanner::FileItemsToAlbums
   std::string songVideoURL; // url to song video
+  std::map<int, ChapterDetails> ChapterMarks; // map of chapter names and start and end times
```

Comment:
Is it intentional to not use the type definition as given in line 52?

The member name should start with `m_` followed by a lower case letter. This rule is violated for the existing members, but for new code we should follow our coding guidelines.

Maybe you wanted to look the line like `ChapterMarks m_chapterMarks;`?

Reply:
Sure. Done.